releasenotes/notes/agent-diagnose-ext-first-cut-0b73b9442cf173e3.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Extended the "agent diagnose" CLI command to allow the easy addition of new
+    diagnostics for diverse and dispersed Agent code.
```

Comment:
I am not sure in this PR there were only a small number and mostly added was a mechanism. Are our release notes more for end users instead of people with dev inclinations.

Reply:
Sure. I can make the change but can we discuss if diagnose all is suitable change by itself or in the longer run we want to change it?

Reply:
Will make the change in the subsequent PR because I will change command CLI